salt/modules/nilrt_ip.py

```diff
@@ -625,7 +626,7 @@ def _enable_dhcp(interface):
     parser.add_section('service_{0}'.format(hwaddr_section_number))
     parser.set('service_{0}'.format(hwaddr_section_number), 'MAC', hwaddr)
     parser.set('service_{0}'.format(hwaddr_section_number), 'Name', 'ethernet_cable_{0}'.format(hwaddr_section_number))
-    parser.set('service_{0}'.format(hwaddr_section_number), 'IPv4.method', 'dhcp')
+    parser.set('service_{0}'.format(hwaddr_section_number), 'IPv4.Method', 'dhcp')
```

Comment:
changing case from method to Method does affect anything? is it intentional?

Reply:
Yes, it's intentional, because Connman recognizes `IPv4.Method` (it's case sensitive)